deploy/vagrant/conf/alluxio.yml

```diff
@@ -4,13 +4,16 @@
 # {Github | Local | Release}
 Type: Local
 
+# If the version is less than 1.1.0, ie. 1.0.1. This should be false for 1.1.0-SNAPSHOT. {true|false}
+Version_LessThan_1.1.0: false
```

Comment:
Yeah, that would work for me. The only caveat is type "Local", but in that case it might okay to assume that the version is at least 1.1.0?